logfebe.go

```diff
@@ -261,11 +261,8 @@ func emitLogRecord(lr *logRecord, sr *serveRecord, target *logplexc.Client,
 		}
 	}
 
-	if sr.Name != "" {
-		// If available, identify what agent is doing the
-		// logging to aid human readers in determining where a
-		// log message came from.
-		msgFmtBuf.WriteString("[" + sr.Name + "] ")
+	if sr.Prefix != "" {
+		msgFmtBuf.WriteString("[" + sr.Prefix + "] ")
```

Comment:
Also, why bracket the prefix in "[...]"? Then it isn't truly a prefix. I was thinking:

```go
  msgFmtBuf.WriteString(sr.Prefix)
```

Basically, I'm saying pg_logplexcollector should get out of this aspect of the formatting game altogether and let the configurer do it all.

You can make whatever argument about how pedantic one would be to not emit a trailing space on the prefix, though, expecting the configurer to also remember to include its own delimiter.